packages/common/container-definitions/src/audience.ts

```diff
@@ -67,4 +104,37 @@ export interface IAudience extends EventEmitter {
 	 * does not technically have a clientId tied to an active connection to the service.
 	 */
 	getMember(clientId: string): IClient | undefined;
+
+	/**
+	 * Returns information about client's connection. Please see {@link ISelf} member descriptions for more details.
+	 * undefined if this client has never connected to the ordering service.
+	 * Please see {@link ISelf.clientId} for more details on when values returned by this function change over time.
+	 *
+	 * @experimental
```

Comment:
Do we have any documented guidance around `@experimental`? Unlike our release tags, this doesn't surface in any meaningful way to consumers. You also note in the remarks that this API is experimental, which may be sufficient for this case, but I wonder if we need to formalize a pattern for adding experimental members to public types. What are our support promises for them / when can we break them? It would be relatively easy to update our API docs gen story to do something with the `@experimental` tag, but I don't know that the intellisense story would be great.

Anyways, not trying to block this or anything, just want to start a discussion 🙂

Reply:
To be honest, I do not know. The API is here to stay, it's the specific behaviors (timing of events) that could change, so I do not want to over-promise here.
I've fired a post on dev channel